src/lib/app-provider/env.ts

```diff
@@ -32,6 +32,7 @@ export enum CELATONE_QUERY_KEYS {
   CODES_BY_WALLET_ADDRESS = "CELATONE_QUERY_CODES_BY_WALLET_ADDRESS",
   CODES_BY_IDS = "CELATONE_QUERY_CODES_BY_IDS",
   CODE_DATA_BY_ID = "CELATONE_QUERY_CODE_DATA_BY_ID",
+  CODE_DATA_INFO = "CELATONE_QUERY_CODE_DATA_INFO",
```

Comment:
remove the old one and change the new one to 
`CODE_DATA = "CELATONE_QUERY_CODE_DATA",`

Reply:
👍🏼